### DISCUSSION TITLE
Beta versions in npm but not in github

### DISCUSSION THREAD

**Discussion by fbritoferreira** [sorry-cypress/cy2#41] [General]
Seems like a few versions got releases recently with the tag v4 but I dont seem them in github are these releases safe?

https://www.npmjs.com/package/cy2?activeTab=versions

**Comment by agoldis** [comment]
You can try them out, but we are still polishing them...

**Reply by agoldis** [reply]
Oh, I see that I published with the `latest` dist-tag

**Reply by fbritoferreira** [reply]
Tried the beta version and seem to be working with cypress 12 is there a ETA on a stable version for it?

**Reply by agoldis** [reply]
@fbritoferreira as soon as I get enough data from users that confirm we've covered all the edge cases. You can help by starting to using beta and reporting issues

**Reply by fbritoferreira** [reply]
@agoldis We have updated to the beta last week and it seems to be working well
